cachito/web/models.py

```diff
@@ -198,7 +198,10 @@ def to_purl(self):
             protocol = match.group("protocol")
             suffix = match.group("suffix")
             has_authority = match.group("has_authority")
-            if not has_authority:
+            if protocol == "file":
+                qualifier = urllib.parse.quote(self.version, safe="")
+                return f"generic/{purl_name}?{qualifier}"
```

Comment:
@mprahl has pointed out that I'm adding a query parameter without `=`. This should probably say something like `f"generic/{purl_name}?file={qualifier}"`

Reply:
The request was successful even without it. So maybe you can just create another PR to update this but there shouldn't be a rush to deploy that right away again? is that correct?